esp/services/ws_cloud/ws_cloudService.cpp

```diff
@@ -149,18 +206,57 @@ bool CWsCloudEx::onGetServices(IEspContext& context, IEspGetServicesRequest& req
     return true;
 }
 
+// Note: metadata.namespace is only included to ensure the hpcc4j regression tests pass.
+// The metadata.labels.app.kubernetes.io/part-of is included to ensure compatibility with
+// older client code targeting versions <1.02. Because of the slash it does not render
+// correctly when converted to PTree and output as XML, which is OK because its not used
+// in the structured XML/SOAP response.
```

Comment:
Is this comment still correct?

Reply:
Yes, I'm including the `part-of` label in the output because the cloud.ts code that parses the json blob will filter out any entries that don't have a label where part-of=HPCC-Platform. The second statement is also true but perhaps not entirely relevant- I'd noticed it only because I was printing the intermediate result as an xml-serialized ptree so I would understand the xpaths to use to query the ptree.